Tutorials/WinForms/Classic/implement-crud-functionality-with-xpo-objects.md

```diff
@@ -0,0 +1,169 @@
+# Implement CRUD functionality with XPO objects
+
+[Step 2 <<](/connect-data-grid-to-xpo-objects.md) 
+&nbsp;&nbsp;&nbsp;
+[Back to TOC](../../)
+&nbsp;&nbsp;&nbsp;
+[>> Step 4](/bind-the-data-grid-to-large-data-source.md)   
+
+## Edit an object in a separate window and save changes
+* Create the *Forms* folder and invoke the context menu for it in the **Solution Explorer**. 
```

Comment:
"invoke the context menu for it" Explain

Reply:
I changed this sentence to "Rigt-click the *Forms* folder in the **Solution Explorer** to open the context menu." Also, changed similar places everywhere.

17ae0b7dedf6ca4b4eca11c2bc984ce16c3f14a2